### ISSUE TITLE
Only q: encrypt stuff, Q: doesn't.

### ISSUE THREAD

**Issue by Reiling-Jeff** [imnaK/quantum#3]
If i accidentally use the **capital letter** of **Q**, it **doesn't encrypt** my text, this may be **bad in some situations**.

Maybe the ability to **encrypt stuff retrospectively** would be a good idea. Like, when i edit my massage and do a "q:" in front of the message when editing, would be a great feature.

**Comment by imnaK** [comment]
Will look into that.

**Comment by Traube1000101** [comment]
Hello Yuuto,

Thank you for your valuable suggestion. I'm pleased to inform you that, I've implemented a second command prefix in the recent commit 6c0d0fbf44f209ef7169b03d40d488d5dabb1cd2. This enhancement allows users to initiate the message with either `q:` or `Q:`, offering an increased level of flexibility.

However, we also evaluated the suggestion of implementing retrospective encryption by editing messages and adding the prefix. After careful consideration, we have decided not to implement this feature. The primary purpose of our message encryption is to prevent sensitive information from being stored on the server. If a message is encrypted after it has been sent, it would have already been saved in its original, unencrypted form on the server, potentially in archives or logs. This would undermine the security benefits of encryption. Therefore, to maintain the integrity and purpose of our encryption feature, we will not be implementing retrospective encryption.
